packages/preview/vienna-tech/1.1.0/template/sections.typ

```diff
@@ -0,0 +1,168 @@
+#import "@preview/rubber-article:0.4.1": ctable
+#import "@preview/vienna-tech:1.1.0": shortcap
+#import "@preview/codly:1.2.0": *
```

Comment:
There is a deprecation warning in this version of codly, and I think it was fixed with v1.3.0. Can you try to update it? For people using your template, it's a better experience if there are no compilation warnings from dependencies.